lite.auto.tfvars.example

```diff
@@ -0,0 +1,254 @@
+
+#-----------------------------------------------------------------------------
+
+# Name of the resource group of the above dns zone.
+# Example entry: "app-dns-prod-eastus2"
+# Value type: string
+# Updateable: N
+
+
+aks_dns_zone_resource_group_name="<dns zone resource group name>"
+
+#------------------------------------------------------------------------------
+
+# The version of HPCC Systems to install.
+# Only versions in nn.nn.nn format the 'latest' are supported.
+# Value type: string
+# Updateable: Y
+
+hpcc_version="9.4.4"
+
+#------------------------------------------------------------------------------
+
+# Enable ROXIE?
+# This will also expose port 8002 on the cluster.
+# Example entry: false
+# Value type: boolean
+# Updateable: Y
+
+aks_enable_roxie=false
+
+#------------------------------------------------------------------------------
+
+# Enable code security?
+# If true, only signed ECL code will be allowed to create embedded language functions, use PIPE(), etc.
+# Example entry: false
+# Value type: boolean
+# Updateable: Y
+
+enable_code_security=false
+
+#------------------------------------------------------------------------------
+
+# If you want a thor cluster then 'enable_thor' must be set to true
+# Otherwise it is set to false
+# Value type: boolean
+# Updateable: Y
+
+enable_thor=true
+
+#------------------------------------------------------------------------------
+
+# The number of Thor workers to allocate.
+# Must be 1 or more.
+# Value type: number
+# Updateable: Y
+
+thor_num_workers=2
+
+#------------------------------------------------------------------------------
+
+# The maximum number of simultaneous Thor jobs allowed.
+# Must be 1 or more.
+# Value type: number
+# Updateable: Y
+
+thor_max_jobs=2
+
+#------------------------------------------------------------------------------
+
+# The amount of storage reserved for the landing zone in gigabytes.
+# Must be 1 or more.
+# If a storage account is defined (see below) then this value is ignored.
+# Value type: number
+# Updateable: Y
+
+storage_lz_gb=25
+
+#------------------------------------------------------------------------------
+
+# The amount of storage reserved for data in gigabytes.
+# Must be 1 or more.
+# If a storage account is defined (see below) then this value is ignored.
+# Value type: number
+# Updateable: Y
+
+storage_data_gb=100
+
+#------------------------------------------------------------------------------
+
+# Map of name => value tags that can will be associated with the cluster.
+# Format is '{"name"="value" [, "name"="value"]*}'.
+# The 'name' portion must be unique.
+# To add no tags, use '{}'.
+# Value type: map of string
+# Updateable: Y
+
+extra_tags={}
+
+#------------------------------------------------------------------------------
+
+# The VM size for each roxie node in the HPCC Systems node pool.
+# Possible values are: large, xlarge, 2xlarge, and 4xlarge
+# Value type: string
+# Updateable: N
```

Comment:
What does #Updateable: N mean?  Does it indicate that I should not change this value?  If that is the case, why is it a variable?